webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require("path");
+
+module.exports = {
```

Comment:
All other frontend packages use rollup.
Why have you decided to go with webpack? If there is any reason, why didn't you convey it to me before proceeding?

Reply:
>Why have you decided to go with webpack?

I initially started with rollup but encountered some errors during bundling. I was attempting to resolve them, but in the meantime, I noticed that webpack is also installed in most of the packages (e.g., [neeto-commons-frontend](https://github.com/bigbinary/neeto-integrations-frontend/blob/main/package.json#L141), [neeto-integrations-frontend](https://github.com/bigbinary/neeto-integrations-frontend/blob/main/package.json#L141)). So, I decided to give it a try, and initially, it was working perfectly fine (posted [testing video here](https://github.com/bigbinary/neeto-hotkeys/pull/4#issue-1990247043)). And then, this issue occured when I changed 'main' to 'dist/index.js'.


> why didn't you convey it to me before proceeding?

This was a lack of communication on my end. I should have discussed with you before choosing webpack. I will keep this in mind.